### PR TITLE
docs: element children theming

### PR DESCRIPTION
**Pull request by friedow**
This adds documentation for theming of elements. This is the documentation for https://github.com/davatorium/rofi/pull/1792. This is part of https://github.com/davatorium/rofi/issues/1817.

### CODE REVIEW

**Comment by DaveDavenport** [comment]
I think this part of the documentation is a better place to modify it:

> A child added with another name is seen as a box, this can be used as dynamic padding to tweak how an entry looks when it expands to available space.

(just before: https://github.com/davatorium/rofi/blob/next/doc/rofi-theme.5.markdown#listview-text-highlight) 

Maybe something like:

A child added with another name is treated the same as the special widget described in the advanced layout section.

(where 'advanced layout' is a link to the advanced layout section anchor).

https://github.com/davatorium/rofi/blob/next/doc/rofi-theme.5.markdown#advanced-layout

**Comment by friedow** [comment]
Good point, this location is much more suitable! I've patched that.

File: doc/rofi-theme.5.markdown [hunk 1]
```diff
@@ -1010,8 +1010,8 @@ By default the `element-icon` and `element-text` child widgets are added to the
 `element`. This can be modified using the `children` property or the
 `[no]-show-icons` option.
 
-A child added with another name is seen as a `box`, this can be used as dynamic
-padding to tweak how an entry looks when it expands to available space.
+A child added with another name is treated the same as the special widget described
+in the [advanced layout](https://github.com/davatorium/rofi/blob/next/doc/rofi-theme.5.markdown#advanced-layout) section.
```
Comment (DaveDavenport) [inline doc/rofi-theme.5.markdown]: Maybe not an absolute link, but relative in current document?

> [advanced layout](#advanced-layout)
```
[advanced layout](#advanced-layout)
```

Comment (friedow) [reply doc/rofi-theme.5.markdown]: Done. I didn't know if the manpage generation could handle relative links ;).

**Pull request merged**